src/Composer/PatchRemoveCommand.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace szeidler\ComposerPatchesCLI\Composer;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use Composer\Json\JsonFile;
+use Composer\Json\JsonManipulator;
+
+class PatchRemoveCommand extends PatchBaseCommand {
+
+  protected function configure() {
+    $this->setName('patch-remove')
```

Comment:
This command is not being added to the README, would be good to document it right away?